lib/models/os.coffee

```diff
@@ -74,6 +74,9 @@ getOsModel = (deps, opts) ->
 			throw new Error("Invalid semver version: #{v}")
 		return vNormalized
 
+	deviceImageUrl = (deviceType, version) ->
```

Comment:
I think `size_estimate` requests need the same fix

https://github.com/resin-io/resin-sdk/blob/e79d43ed47aa423403ff5b3f55fe47bc19a8567e/lib/util/img-maker.coffee#L27
usage example: https://github.com/resin-io/resin-sdk/blob/e79d43ed47aa423403ff5b3f55fe47bc19a8567e/lib/models/os.coffee#L45